src/ui/public/field_format_editor/editors/url/url.html

```diff
@@ -7,6 +7,13 @@
   </select>
 </div>
 
+<div class="checkbox" ng-if="editor.formatParams.type == 'a'">
+  <label>
+    <input ng-model="editor.formatParams.openLinkInCurrentTab" class="ng-pristine ng-valid ng-touched" type="checkbox">
```

Comment:
These attributes should be one per line. See [here](https://github.com/fbaligand/kibana/blob/master/style_guides/html_style_guide.md#multiple-attribute-values) for details.